third_party/boringssl/src/crypto/internal.h

```diff
@@ -146,6 +157,7 @@
 #endif
 #endif
 
+#if !defined(IS_COBALT_HERMETIC_BUILD)
```

Comment:
There's a section here that includes pthread already, and sets OPENSSL_PTHREAD too. Maybe we should allow this to compile and remove the other `#include <pthread.h>` from the section above ?